log/format.go

```diff
@@ -79,12 +79,11 @@ type TerminalStringer interface {
 // a terminal with color-coded level output and terser human friendly timestamp.
 // This format should only be used for interactive programs or while developing.
 //
-//     [LEVEL] [TIME] MESSAGE key=value key=value ...
+//	[LEVEL] [TIME] MESSAGE key=value key=value ...
 //
 // Example:
 //
-//     [DBUG] [May 16 20:58:45] remove route ns=haproxy addr=127.0.0.1:50002
-//
+//	[DBUG] [May 16 20:58:45] remove route ns=haproxy addr=127.0.0.1:50002
 func TerminalFormat(usecolor bool) Format {
```

Comment:
Does this also update the logs in the file or just in the terminal? I guess they don't really print to the terminal, but just to be sure

Reply:
it does. Seems to me that this is a case where the function name does not match what it actually does, because geth.log content was succesfully modified by this change.